tests/security/SingleParticipantWithSecurity/SingleParticipantWithSecurity.mpc

```diff
@@ -0,0 +1,8 @@
+project: dcps_cm, dcps_rtps_udp, opendds_optional_security {
```

Comment:
Already commented this, but it was at a bad time. Change to `opendds_security` because it requires security and the non-security builds are failing to build this test.

```suggestion
project: dcps_cm, dcps_rtps_udp, opendds_security {
```

Reply:
Sorry, missed that